render/src/shape_utils.rs

```diff
@@ -1222,6 +1295,31 @@ pub fn quadratic_curve_bounds(
         ))
 }
 
+pub fn cubic_curve_bounds(
+    start: swf::Point<Twips>,
+    stroke_width: Twips,
+    control_a: swf::Point<Twips>,
+    control_b: swf::Point<Twips>,
+    anchor: swf::Point<Twips>,
+) -> Rectangle<Twips> {
+    // [NA] Should we just move most of our math in this file to lyon_geom?
```

Comment:
Honestly, I can't make up my mind on what to think about this... Many pros and cons on both sides...